tests/plugins/add-to-triage-project/index.js

```diff
@@ -60,5 +60,36 @@ describe("add-to-triage-project", () => {
 
             expect(addIssueToTriageProject.isDone()).toBeTruthy();
         });
+
+        test("doesn't add the issue to the project when 'triage:no' label is present", async() => {
+
+            // no nock because we don't want any API calls
```

Comment:
I think we should use nock here even though we're not expecting any API calls, so that this test fails in nock instead of on a real network request.

Reply:
Done